tests/plotting/test_widgets.py

```diff
@@ -10,6 +10,7 @@
 from pyvista.plotting.affine_widget import DARK_YELLOW
 from pyvista.plotting.affine_widget import get_angle
 from pyvista.plotting.affine_widget import ray_plane_intersection
+from tests.conftest import flaky_test
```

Comment:
Not sure if the decorator should be defined in `conftest`, or elsewhere...

Reply:
Fine to define in `conftest` .